front/tailwind.config.ts

```diff
@@ -11,5 +20,14 @@ export default {
   theme: {
     extend: {},
   },
-  plugins: [heroui()],
+  plugins: [heroui({
+    themes: {
+      dark: {
+        colors: darkColors,
```

Comment:
maybe: 
```suggestion
     colors: {...semanticColors.dark, DEFAULT: '#fff'},
```
Instead of mutating the object :man_shrugging: